### ISSUE TITLE
Better variable names?

### ISSUE THREAD

**Issue by voliminal991** [jerometwell/sonicwall_parse#11]
Hi Guys,

I got confused in the method "parse_objects" because we have key, .keys(), parsed_key, objects, object_id, object_type, object_property etc. 

I really struggled to work out exactly what was going on to begin with, and found it really difficult to work out why stuff wasn't working! 

I don't know what you think?

David

**Comment by jerometwell** [comment]
Yes, could be clearer! Originally was only parsing "objects" with internal
IDs, but now the vocabulary range has improved we should clear this up.
